### PR TITLE
Added new mixins

### PR DESCRIPTION
**Pull request by null-none**
Added NeverCacheMixin, CacheMixin and CacheControlMixin.

### CODE REVIEW

**Comment by null-none** [comment]
Added timezone mixin

File: mixin/views/_other.py [hunk 1]
```diff
@@ -8,6 +11,80 @@
     from django.urls import resolve
 except ImportError:
     from django.core.urlresolvers import resolve
+import pytz
+
+
+class PaginatorMixin(object):
```
Comment (auvipy) [inline mixin/views/_other.py:17]: object is not needed

File: mixin/views/_other.py [hunk 2]
```diff
@@ -8,6 +11,80 @@
...
+            queryset_paginated = paginator.page(self.request_page)
+        except PageNotAnInteger:
+            queryset_paginated = paginator.page(1)
+        except EmptyPage:
+            queryset_paginated = paginator.page(paginator.num_pages)
+
+        return queryset_paginated
+
+
+class TimezoneMixin(object):
```
Comment (auvipy) [inline mixin/views/_other.py:53]: object no needed

File: mixin/views/_other.py [hunk 3]
```diff
@@ -8,6 +11,80 @@
...
+    def process_request(self, request):
+        tzname = request.session.get('django_timezone')
+        if tzname:
+            timezone.activate(pytz.timezone(tzname))
+        else:
+            timezone.deactivate()
+
+
+class NeverCacheMixin(object):
```
Comment (auvipy) [inline mixin/views/_other.py:62]: object not needed

File: mixin/views/_other.py [hunk 4]
```diff
@@ -8,6 +11,80 @@
...
+    @method_decorator(never_cache)
+    def dispatch(self, *args, **kwargs):
+        return super(NeverCacheMixin, self).dispatch(*args, **kwargs)
```
Comment (auvipy) [inline mixin/views/_other.py:65]: only super() should be ok

**Review by auvipy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**